bedrock/firefox/templates/firefox/nightly-firstrun.html

```diff
@@ -15,85 +15,65 @@
+      <h2 class="contribute-title">{{ ftl('nightly-firstrun-choose-an-area') }}</h2>
       <ul class="mzp-l-card-third mzp-t-dark">
         {% call picto_card(
-          title=_('Test'),
+          title=ftl('nightly-firstrun-test'),
           class='test',
           heading_level=3,
           custom_desc=True
         ) %}
-          <p>{{ _('Find and file bugs and generally make sure things work as they should.') }}</p>
-          <a href="https://quality.mozilla.org/get-involved/" class="mzp-c-button mzp-t-product mzp-t-dark" data-cta-type="button" data-cta-text="Start testing">{{ _('Start testing') }}</a>
+          <p>{{ ftl('nightly-firstrun-find-and-file-bugs') }}</p>
+          <a href="https://quality.mozilla.org/get-involved/" class="mzp-c-button mzp-t-product mzp-t-dark" data-cta-type="button" data-cta-text="Start testing">{{ ftl('nightly-firstrun-start-testing') }}</a>
         {% endcall %}
 
         {% call picto_card(
-          title=_('Code'),
+          title=ftl('nightly-firstrun-code'),
           class='code',
           heading_level=3,
           custom_desc=True
         ) %}
-          <p>{{ _('File bugs and work on the building blocks of the Firefox browser.') }}</p>
-          <a href="https://developer.mozilla.org/docs/Introduction" class="mzp-c-button mzp-t-product mzp-t-dark" data-cta-type="button" data-cta-text="Start coding">{{ _('Start coding') }}</a>
+          <p>{{ ftl('nightly-firstrun-file-bugs-and-work') }}</p>
+          <a href="https://developer.mozilla.org/docs/Introduction" class="mzp-c-button mzp-t-product mzp-t-dark" data-cta-type="button" data-cta-text="Start coding">{{ ftl('nightly-firstrun-start-coding') }}</a>
         {% endcall %}
 
         {% call picto_card(
-          title=_('Localize'),
+          title=ftl('nightly-firstrun-localize'),
           class='localize',
           heading_level=3,
           custom_desc=True
         ) %}
-          <p>{{ _('Make Firefox available (and better) in more languages around the world.') }}</p>
-          <a href="{{ _('https://wiki.mozilla.org/L10n:Contribute') }}" class="mzp-c-button mzp-t-product mzp-t-dark" data-cta-type="button" data-cta-text="Start localizing">{{ _('Start localizing') }}</a>
+          <p>{{ ftl('nightly-firstrun-make-firefox-available') }}</p>
+          <a href="{{ ftl('nightly-firstrun-contribute-link') }}" class="mzp-c-button mzp-t-product mzp-t-dark" data-cta-type="button" data-cta-text="Start localizing">{{ ftl('nightly-firstrun-start-localizing') }}</a>
         {% endcall %}
       </ul>
     </div>
   </section>
-
-  {% if not request.locale.startswith('en') %}
-    {% if l10n_has_tag('promo_locale') %}
-    {# The following block is entirely defined by locales, and the placeholder in en-US here will never be exposed to production #}
-    <div class="mzp-l-content">
-      <div class="locale-specific">
-        {% l10n promo_locale 20150101 %}
```

Comment:
@pmac I don't think we have an equivalent system for this in Fluent, right?

Reply:
these were really just localized templates managed in the l10n repo. So we kindof do with l10n templates that we could manage in bedrock if we want/need to keep any of this, it just wouldn't be managed by the l10n team.

Reply:
@peiying2 is this content you would like to keep? We'd need to manage the content in bedrock when this page transitions to Fluent, which is something I can take care of here, but it would probably be less visible / accessible to the L10n community.

It looks like ~6 locales have custom content: https://github.com/mozilla-l10n/www.mozilla.org/search?p=1&q=promo_locale

Reply:
OK, I'm going to move this content into bedrock for the time being, just so it's not lost.